### PR TITLE
feat: added simple navigation

### PR DESCRIPTION
**Pull request by bhavyaj12**
Added the following:
- Simple Navigation 
- Page is responsive, please check this
- Preview: https://deploy-preview-13--halcyon-ui.netlify.app/pages/components/navigation

Nav simple

### CODE REVIEW

File: pages/components/navigation.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="HTML, CSS, Component Library, UI, UX" />
+    <meta name="description" content="Stylish pure CSS Component Library for making beautiful UI fast." />
+    <meta name="author" content="Bhavya Joshi" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/nav-common.css">
+    <link rel="stylesheet" href="/css/utility.css">
+    <link rel="stylesheet" href="/css/badge.css">
+    <link rel="stylesheet" href="/css/navigation.css">
```
Comment (krgarima) [inline pages/components/navigation.html:16]: You can import these in a single CSS file and import that file only.

File: pages/components/navigation.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            style="width: 849px; height: 743px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (krgarima) [inline pages/components/navigation.html:130]: Inline styling can be avoided here.

**Review by krgarima — COMMENTED** [review]
Some minor suggestions to follow. Rest looks good to me.

**Review by rishichourasia — APPROVED** [review]
Garima have listed suggested changes , other than that looks good to merge .

**Pull request merged**